<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>211</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #333333;
    border: 2px solid #555555;
    border-radius: 5px;
    padding: 8px;
    font-size: 16px;
    color: #FFFFFF;
}

QLineEdit:hover {
    border: 2px solid #777777;
}

QLineEdit:focus {
    border: 2px solid #2574F5;
    background-color: #222222;
}
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #333333;
    border: 2px solid #555555;
    border-radius: 5px;
    padding: 8px;
    font-size: 16px;
    color: #FFFFFF;
}

QLineEdit:hover {
    border: 2px solid #777777;
}

QLineEdit:focus {
    border: 2px solid #2574F5;
    background-color: #222222;
}
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2574F5;  /* Normal Blue */
    color: white;
    border-radius: 10px;
    border: 2px solid #1E5DC2;  /* Slightly darker border */
    padding: 10px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #4590FF;  /* Lighter blue on hover */
    border: 2px solid #1B52A5;
}

QPushButton:pressed {
    background-color: #1E5DC2;  /* Darker blue when pressed */
    border: 2px solid #163F87;
}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>130</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #505050;  /* Dark Grey */
    color: white;
    border-radius: 10px;
    border: 2px solid #404040;  /* Slightly darker border */
    padding: 10px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #606060;  /* Lighter dark grey on hover */
    border: 2px solid #505050;
}

QPushButton:pressed {
    background-color: #303030;  /* Even darker grey when pressed */
    border: 2px solid #202020;
}
</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="InvalidLoginLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>Invalid username or password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guestButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #5C7893;
    color: white;
    border-radius: 10px;
    border: 2px solid #486076;
    padding: 10px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #6E8AA4;
    border: 2px solid #3F5263;
}

QPushButton:pressed {
    background-color: #4E657D;
    border: 2px solid #354755;
}
</string>
   </property>
   <property name="text">
    <string>Login as guest</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerMenuButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #5C7893;
    color: white;
    border-radius: 10px;
    border: 2px solid #486076;
    padding: 10px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #6E8AA4;
    border: 2px solid #3F5263;
}

QPushButton:pressed {
    background-color: #4E657D;
    border: 2px solid #354755;
}
</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="emptyFieldsLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>Fields cannot be empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>631</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: White</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:96pt; font-weight:700;&quot;&gt;Astro Nav&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>591</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(90,90,90,0.5);
border: 2px solid #2574F5;
border-radius: 35px;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>usernameLineEdit</zorder>
  <zorder>passwordLineEdit</zorder>
  <zorder>loginButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>InvalidLoginLabel</zorder>
  <zorder>guestButton</zorder>
  <zorder>registerMenuButton</zorder>
  <zorder>emptyFieldsLabel</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
